_posts/tutorial/2022-07-07-radiuss-on-aws.md

```diff
@@ -1,36 +1,26 @@
 ---
 layout: post
-title: "RADIUSS AWS Tutorials - August 2022"
+title: "RADIUSS AWS Tutorials: August 2022"
 category: event
 tags: [tutorial, event]
 ---
 
-Come and learn how to use some LLNL software in the cloud this summer. In
-August, we will be hosting a series of tutorials in collaboration with AWS,
-showing you how to install and use these projects on AWS EC2 instances. No
-previous experience necessary, sign up and learn with us.
+## Learn how to use a modern HPC software stack
 
-[Sign Up!](https://forms.gle/3wAKK5PBizUiNaVg6)
+Join us this summer as we host a series of tutorials in collaboration with AWS, demonstrating how to use several GPU-ready projects in the cloud and on premises. Follow along on your own EC2 instance (provided). No previous experience necessary.
 
-# Schedule
+Collaborate with us on [**RADIUSS Slack**](https://radiuss-llnl.slack.com) and [**download the flier**](/radiuss/assets/files/RADIUSS_tutorials_flier_final.pdf) (127 KB PDF) to share with your colleagues.
 
-- _Week of 8/1/2022_: Spack
-- _8/8/2022_: BLT 
-- _8/10/2022_: Caliper
-- _8/15/2022_: MFEM
-- _Week of 8/25/2022_: Ascent
-- _8/25/2022_: RAJA
+## We're offering seven tutorials in August
 
-# Projects
+[![Sign up button](/radiuss/assets/img/posts/tutorials-sign-up-button.png)](https://forms.gle/3wAKK5PBizUiNaVg6)
 
-- [Spack](https://github.com/spack/spack)
-- [Caliper](https://github.com/LLNL/Caliper)
-- [BLT](https://github.com/LLNL/blt)
-- [Flux](https://github.com/LLNL/Caliper)
-- [MFEM](https://github.com/mfem/mfem)
-- [Ascent](https://github.com/alpine-dav/ascent)
-- [RAJA](https://github.com/LLNL/RAJA)
-
-# Resources
-
-- [RADIUSS Slack](https://radiuss-llnl.slack.com)
+| Date | Deadline to sign up | Project |
+| ---- | ------------------- | ------- |
+| Week of August 1, 2022 | July 25 | ![Spack logo](/radiuss/assets/img/posts/llnl-spack.png) Learn to install your software quickly with [**Spack**](https://github.com/spack/spack) |
```

Comment:
Let's have these organized with the assets/posts, in the same directory with the pdf. And for any URL that you use, it should start with `{{ site.baseurl }}` otherwise it will 404. We should generally not hard code the baseurl (radiuss) into the markdown, as it should work out of the box if someone decided to change it.

Reply:
I'm also wondering - if we already are using these logos on the software page: https://software.llnl.gov/radiuss/projects/ should we be redundantly adding them here? You can look at that page to see how to look them up in the site data.

Reply:
I copied these 7 logos into their own folder b/c I had to size them down so they'd look decent in the table; I didn't want to mess up however they were already being used. They are all 75 px wide.

I'm working on a commit w/ the button and file path changes. Thanks for the feedback, as I wasn't sure about where to put the new files.

Reply:
ah, that makes sense! Let's make an assets folder just for small logos then, and we can use them consistently across future posts (and this on).

Reply:
Heads up I'm going into pair programming so I might fall off the thread for a bit - I promise I'll get it reviewed after the work day at the latest!

Reply:
New button, color matches links on the page:
![image](https://user-images.githubusercontent.com/31740953/179603246-6bef42cf-c463-4b27-9b60-d437a70f7558.png)

Rollover state:
![image](https://user-images.githubusercontent.com/31740953/179603435-b284f923-6f31-4734-9669-a35ac492e53a.png)

Reply:
> ah, that makes sense! Let's make an assets folder just for small logos then, and we can use them consistently across future posts (and this on).

New commits address everything except for this. Stay tuned :)